streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -373,7 +374,7 @@ class StreamingContextSuite extends SparkFunSuite with BeforeAndAfter with TimeL
     Thread.sleep(100)
   }
 
-  test ("registering and de-registering of streamingSource") {
+  test("registering and de-registering of streamingSource") {
```

Comment:
Usually, we don't touch the other test cases because it makes backporting this PR difficult.
Please revert this and focus on your test case.

Reply:
Got it, reverted.